### PR TITLE
tests: Redo waiting for elements to be removed to resolve warning

### PR DESCRIPTION
**Pull request by atodorov**
gets rid of:

Warning: You seem to have overlapping act() calls, this is not
supported. Be sure to await previous act() calls before making a
new one.

### CODE REVIEW

**Review by Gundersanne — APPROVED** [review]

File: src/test/SmartComponents/CreateImageWizard/CreateImageWizard.test.js [hunk 1]
```diff
@@ -258,16 +258,19 @@ describe('Step Registration', () => {
             .getByLabelText('Embed an activation key and register systems on first boot')
             .click();
 
-        const p1 = waitForElementToBeRemoved(() => screen.queryByTestId('organization-id'));
-        const p2 = waitForElementToBeRemoved(() => screen.queryByTestId('subscription-activation'));
+        const p1 = waitForElementToBeRemoved(() => [
+            screen.getByTestId('organization-id'),
+            screen.getByTestId('subscription-activation'),
+        ]);
+        // const p2 = waitForElementToBeRemoved(() => screen.queryByTestId('subscription-activation'));
```
Comment (Gundersanne) [inline src/test/SmartComponents/CreateImageWizard/CreateImageWizard.test.js:265]: Just remove this line

File: src/test/SmartComponents/CreateImageWizard/CreateImageWizard.test.js [hunk 2]
```diff
@@ -258,16 +258,19 @@ describe('Step Registration', () => {
...
 
         // then click the first radio button which should remove any input fields
         screen
             .getByTestId('register-later-radio-button')
             .click();
 
         await p1;
-        await p2;
+        // await p2;
```
Comment (Gundersanne) [inline src/test/SmartComponents/CreateImageWizard/CreateImageWizard.test.js:273]: and this one, no need to comment them out, just remove

**Review by Gundersanne — CHANGES_REQUESTED** [review]
Thank you! Just remove the lines tbh, no need to comment them out.

**Comment by atodorov** [comment]
> Thank you! Just remove the lines tbh, no need to comment them out.

Thanks for catching this. All updated.

**Review by Gundersanne — APPROVED** [review]
ay nice, thanks!

**Pull request merged**